components/builder-depot/src/server.rs

```diff
@@ -1216,10 +1208,21 @@ fn create_channel(req: &mut Request) -> IronResult<Response> {
         };
     }
 
+    let origin_id = match try!(get_origin(req, &origin)) {
+        Some(origin) => {
+            origin.get_id()
+        }
+        None => {
+            error!("Origin {} not found!", origin);
```

Comment:
This should be `info` or `debug` level. You can't see this now because I've done a bad job at documenting my intentions and we haven't hooked our logs into an aggregator/analyzer just yet, but the `error` log level will be something that wakes you up in the middle of the night if you're on call. `warn` is something that will send you an alert during core hours unless *a lot* of repetitive warnings come in